### DISCUSSION TITLE
sbt 0.13 shell syntax warning does not include information in which line of code the problem is located

### DISCUSSION THREAD

**Discussion by nemoo** [sbt/sbt#6819] [Q&A] [answered]
My build on sbt 1.6.0 currently produces a warning that I would like to get rid of:
`[warn] sbt 0.13 shell syntax is deprecated; use slash syntax instead: IntegrationTest / test`

I guess the relevant piece of the build.sbt file is: 

```// define a new configuration for integration tests
lazy val ITest = config("it") extend(Test)

lazy val root = (project in file("."))
  .enablePlugins(PlayScala)
  .configs(ITest)
  .settings( inConfig(ITest)(Defaults.testSettings),
    ITest / scalaSource := baseDirectory.value / "it",
    ITest / testOptions += Tests.Argument("-oF"))  // show full stacktrace https://stackoverflow.com/questions/7237824/how-can-i-get-complete-stacktraces-for-exceptions-thrown-in-tests-when-using-sbt
```

But I cannot find where I use sbt 0.13 shell syntax, and sbt also does not tell me where exaclty it found the deprecated syntax.

I hope it is not this line: `lazy val root = (project in file("."))` because that is taken right out of the documentation.

So maybe someone can give me a hint? Or, even better maybe sbt could point to the exact location in the future?

**Comment by SethTisue** [comment, marked as answer]
Is the problem reproducible without involving the `PlayScala` plugin? Perhaps the root cause is there. If I comment out that line, I don't see any warning.

**Reply by mzuehlke** [reply]
The [PR](https://github.com/playframework/playframework/pull/11121) to remove the slash syntax from the `PlayScala` plugin is very recent and not in any release, yet.

It would be good if the warning would only be triggered, if the "faulty code" is in the current project and not in a plugin.

**Reply by nemoo** [reply]
So it will be fixed in the next play framework release. Sounds good, closing this.
